src/tool/subcommands/api_cmd.rs

```diff
@@ -50,9 +51,20 @@ pub enum ApiCommands {
         #[arg(long, value_enum, default_value_t = RunIgnored::Default)]
         /// Behavior for tests marked as `ignored`.
         run_ignored: RunIgnored,
+        /// Maximum number of concurrent requests
+        #[arg(long, default_value = "8")]
+        max_concurrent_requests: usize,
```

Comment:
Supporting a way to turn off concurrent requests limitation could be nice (i.e., using the exceptional value 0). But for now I think this is ok.